packages/styles/scss/patterns/blocks/listsection/_listsection.scss

```diff
@@ -76,6 +76,17 @@
       }
     }
 
+    .#{$prefix}--content-group__title {
+      padding-left: 0;
```

Comment:
@RaphaelAmadeu I'm seeing a lot of this in the `UseCases` pattern I'm currently refactoring. Styles like this aren't really necessary when using Carbon's grid. I've been removing stuff like this wherever I see it.

Reply:
👆